nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -116,6 +117,28 @@ Copyright (C) Microsoft Corporation. All rights reserved.
             </ClCompile>
         </ItemGroup>
     </Target>
+  
+    <!-- Fix includes to point to the generated C++/WinRT headers rather than the one in the SDK -->
+    <Target Name="CppWinRTFixSDKIncludes"
+            BeforeTargets="SetBuildDefaultEnvironmentVariables"
+            Condition="'$(CppWinRT_IncludePath)' != '$(CppWinRT_OldIncludePath)'">
+        <ItemGroup>
+            <_FixedSDKIncludes Remove="@(_FixedSDKIncludes)"/>
+            <_FixedSDKIncludes Include="$(WindowsSDK_IncludePath)"/>
+            <_FixedSDKIncludes Remove="$(CppWinRT_OldIncludePath)"/>
+            <_FixedSDKIncludes Include="$(CppWinRT_IncludePath)"/>
+        </ItemGroup>
+        <ItemGroup>
+            <_FixedIncludes Remove="@(_FixedIncludes)"/>
+            <_FixedIncludes Include="$(IncludePath)"/>
+            <_FixedIncludes Remove="$(CppWinRT_OldIncludePath)"/>
```

Comment:
Does this actually work since IncludePath is a property? I don't think there is magic property to itemgroup conversion is there? So if my include path Contains multiple paths this wouldn't work?

Reply:
There is actually a magic semi-colon separated list to item group conversion and vice versa which Scott informed me of.  See it below in action.

![image](https://user-images.githubusercontent.com/1691134/68443100-ec9ac480-0187-11ea-80d7-db2e61cbb930.png)